Wikimate.php

```diff
@@ -2075,7 +2088,7 @@ public function getUser($revision = null)
 
     /**
      * Returns the ID of the user who uploaded this file,
-     * or of its specified revision.
```

Comment:
How about "...or a given revision of it, if specified"? (Same comment for line 2055).

Reply:
Uhm, that phrase occurs 19 times in total. I agree your proposal is more precise, but then it would apply to a lot more than 2 lines. Is that what you actually intended?

Reply:
I'd prefer that, yes. I didn't realize it was so common a phrase, but since you're already making changes across the board in the docs, I don't feel like it's expanding the scope of this PR too much.

Reply:
Done. Didn't rebase, the commit can standalone.

Reply:
Agreed.